app/controllers/v1/interventions_controller.rb

```diff
@@ -2,10 +2,12 @@
 
 class V1::InterventionsController < V1Controller
   def index
+    starred_interventions_ids = current_v1_user.stars.pluck(:intervention_id)
+
     collection = interventions_scope.detailed_search(params, current_v1_user)
-    paginated_collection = V1::Paginate.call(collection, start_index, end_index)
+    collection = collection.sort { |a, b| starred_interventions_ids.count(b.id) <=> starred_interventions_ids.count(a.id) }
```

Comment:
I'm afraid it isn't efficient. Could you tell me the main difference between `order` and `sort`? It will be awesome to order this in one query 🚀